js/pulse.js

```diff
@@ -197,12 +198,18 @@ var PulseMaker = {
   'toggleAdditionalFeatures' : function () {
     if (FEATURE.orphans) { typography.preventTextOrphans(); }
   },
+  generatePulseProjectId(projectData) {
+    var timestamp = projectData.Timestamp ? projectData.Timestamp : false;
+    var projectId = timestamp ? Date.parse(timestamp) : '0';
```

Comment:
> Will all projects have a unique time stamp? What about batch import projects (we might just migrate all hive portfolio projects at once)?

@xmatthewx and I have been trying to figure out a way to generate unique ID for spreadsheet data (we have this issue with the event app too)... and we found timestamp being the most reliable/safe field to generate ID from. 

> More of a design suggestion, but this isn't a very readable in the url. Can we use a slug version of the title + maybe append a timestamp? Something like what Medium does?

I'm not sure how often project title gets updated, and if any changes happen in spreadsheet will it update the timestamp as well? @xmatthewx